examples/step-26/step-26.cc

```diff
@@ -260,16 +260,17 @@ namespace Step26
     laplace_matrix.reinit(sparsity_pattern);
     system_matrix.reinit(sparsity_pattern);
 
+    // do not pass the ConstraintMatrix as the last argument to the following
+    // functions, because our constraints are changing over time and are
+    // condense()-ed later.
```

Comment:
Given that this is a tutorial program, can you put this into the comment above this block with some more explanation?